ImageClassification/ViewControllers/ViewController.swift

```diff
@@ -760,6 +769,28 @@ extension ViewController {
         print("value is", Int(sender.value))
         speechSpeedDegree = sender.value
     }
+    
+    func areaBoundSetup(){
+        previewView.addSubview(areaBound)
+        areaBound.translatesAutoresizingMaskIntoConstraints = false
+        areaBound.centerYAnchor.constraint(equalTo: view.centerYAnchor).isActive = true
+        areaBound.centerXAnchor.constraint(equalTo: view.centerXAnchor).isActive = true
+        areaBound.widthAnchor.constraint(equalToConstant: view.frame.size.width-10).isActive = true
+        areaBound.heightAnchor.constraint(equalToConstant: view.frame.size.width-10).isActive = true
+        areaBound.layer.borderWidth = 2
+        areaBound.layer.borderColor = UIColor.red.cgColor
+        
+        let textView = UILabel()
+        areaBound.addSubview(textView)
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.topAnchor.constraint(equalTo: areaBound.topAnchor, constant: 2).isActive = true
+        textView.leadingAnchor.constraint(equalTo: areaBound.leadingAnchor, constant: 2).isActive = true
+        textView.heightAnchor.constraint(equalToConstant: 12).isActive = true
+        textView.text = "Nothing Detected"
+        textView.font = UIFont.systemFont(ofSize: 12)
+        textView.textColor = .red
+    }
```

Comment:
I think this is interesting and we should experiment with it more. Do not forget to document this!